newrelic-python.ini

```diff
@@ -61,32 +66,11 @@ security.mode = RASP
 security.validator_service_endpoint_url = ws://your_validator_system_ip:54321
```

Comment:
security.validator_service_endpoint_url should be security.validator_service_url

Reply:
Updated